### ISSUE TITLE
case command line argument being changed when command has trailing space

### ISSUE THREAD

**Issue by TheFlyingBadger** [microsoft/terminal#12612]
### Windows Terminal version

1.11.3471.0

### Windows build number

10.0.19042.1526

### Steps to reproduce

I have a ssh profile connecting to a unix box.

the command line is 
`ssh beast-debian `

"beast-debian" is an alias in %USERPROFILE%\.ssh\config

```
Host beast-debian
  User badger
  HostName 10.1.1.110
  ProxyCommand ssh -W %h:%p sshJump
  LocalForward 13389 10.1.1.110:3389
```

however, when I update the command line within terminal to remove (for instance " -v") extra parameters, it's updating the  ssh alias to be `beast-Debian` which is NOT in my ssh config file

### Expected Behavior

do not randomly change the name of my alias.

it only seems to happen if I leave a trailing space on the end of the command

`ssh beast-debian ` (with trailing space)
becomes
`ssh beast-Debian `

but 
`ssh beast-debian`
is fine

### Actual Behavior

keeps changing the value to be beast-Debian

**Comment by zadjii-msft** [comment]
Weird. Which shell are you using? `cmd`, `powershell`, `pwsh`, `bash`, something else...?

**Comment by zadjii-msft** [comment]
Wait so like, from the Run Dialog? From the Start Menu? Maybe grab a screen recording with something like ScreenToGif?

**Comment by TheFlyingBadger** [comment]
![terminal](https://user-images.githubusercontent.com/40585882/157573768-b0e179e9-3af9-4072-83a2-44efaf153eb8.gif)


Here's a gif of my adding a trailing space to the SSH command and the "autocorrect" being applied...

**Comment by ianjoneill** [comment]
This was fixed in #11137 - so this will be included in the 1.12 release when it becomes available in the Windows store (or you can download it from the releases page).

**Comment by DHowett** [comment]
Thanks for the report! Yup, looks like a /duplicate of #11133.
